db/indexed_docs_test.go

```diff
@@ -94,11 +94,11 @@ func (b *indexKeyBuilder) Col(colName string) *indexKeyBuilder {
 	return b
 }
 
-// Field sets the field name for the index key.
+// Fields sets the fields' names for the index key.
```

Comment:
nitpick: I think `fields' names` implies that each field has multiple names.  If each field only has one name then `field names` is more appropriate.  Same goes for the variable names below.

Reply:
adjusted